pkgs/development/tools/testkube/default.nix

```
@@ -0,0 +1,31 @@
+{ buildGoModule, fetchFromGitHub, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "testkube";
+  version = "1.10.40";
+
+  src = fetchFromGitHub {
+    owner = "kubeshop";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-6U9tbn1hp4OEI5ZNcknXBm64nJtZtC9nuIpsHSl1rQw=";
+  };
+
+  subPackages = [ "cmd/kubectl-testkube/main.go" ];
+
+  vendorSha256 = "sha256-DLh4uKT4/YbA0ZjEZXmryyB5BnlL2XD+8JvHy+Su6pM=";
+
+  ldflags = [ "-s" "-w" ];
+
+  postInstall = ''
+    mv $out/bin/main $out/bin/kubectl-testkube
+  '';
```

```suggestion
  postInstall = ''
    mv $out/bin/main $out/bin/kubectl-testkube
    installShellCompletion --cmd kubectl-testkube \
      --bash <($out/bin/kubectl-testkube completion bash) \
      --fish <($out/bin/kubectl-testkube completion fish) \
      --zsh <($out/bin/kubectl-testkube completion zsh)
  '';
```
Add shell completion.